### PR TITLE
Image features added 

### PR DESCRIPTION
**Pull request by ishimwepati**
This project has a lot functionalities :
1. Images have been used,
2. Alignement using CSS
3. div tag has been used.

### CODE REVIEW

**Comment by ishimwepati** [comment]
Hello [@ishimwepati ] 👋 ,

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!
unnamed

Kindly work on the issues below to finish and pass to the next project.

**Comment by ishimwepati** [comment]
kindly use a div tag as a container of the hamburger for convention, semantics, and user experience(UX) 😄

Screenshot 2023-06-13 at 19 57 44

**Comment by ishimwepati** [comment]
Kindly use flex property to align your header items as it helps with the flexibility of your nav items 😄

In index.html at line 38 to 128.

**Comment by ishimwepati** [comment]
Please the container of this headline section should be a section tag and not a nav tag as it represents a generic standalone section of your project 😄

**Comment by ishimwepati** [comment]
Optional : 

1. Make efforts in using linters in your next project
2. Apply HTML and CSS best practice

**Comment by ishimwepati** [comment]
I suggest that you document well your ReadMe.md file. 😄

In

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @ishimwepati, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
almost

## Highlights

- [ ] Github Feaure Fairly used :green_circle: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please ensure that you follow this [step](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) to know how to use Review Features in Github as required by the project requirement, especially inline comment and the commenting section.  GitHub's review features provide a structured and efficient workflow for conducting code reviews. Reviewers can add comments, suggest changes, approve or request changes to specific lines or sections of code. This makes the review process more focused, organized, and easily trackable, leading to faster iterations and better collaboration among team members. :slightly_smiling_face: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -106,6 +94,38 @@ <h2 class="project-tile">These are some of our project </h2>
...
+            <span class="code">&lt;</span>
+            Map
+            <span class="code">&#47;&gt;</span>
+          </p>
+        </a>
+        <a
+          href="#"
+          target="_blank"
+          class="project-tile"
+        >
```
Comment (ishimwepati) [inline index.html:117]: Kindly use a div tag as a container of the hamburger for convention, semantics, and user experience(UX), it makes your code clean😄

File: index.html [hunk 2]
```diff
@@ -36,11 +24,11 @@
...
     <section id="welcome-section">
-      <h1>Hey we are Developers</h1>
+      <h1>Hello, I'm a developper</h1>
     </section>
 
     <section id="projects">
-      <h2 class="project-tile">These are some of our project </h2>
+      <h2 class="project-tile">These are some of our projects </h2>
       <div class="projects-grid">
     <a
```
Comment (ishimwepati) [inline index.html:33]: Please use flex property to align your header items as it helps with the flexibility of your nav items 😄

**Review by ishimwepati — COMMENTED** [review]
Hello [@ishimwepati ] 👋 ,

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

unnamed

**Review by ishimwepati — COMMENTED** [review]
I suggest that you document well your ReadMe.md file by following the https://github.com/microverseinc/readme-template model. 😄

**Review by AbelG101 — APPROVED** [review]
# Hi @ishimwepati 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Proper use of GitHub inline comments feature ✔
- Comments are phrased politely ✔
- Comments are clear and easy to understand ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**